inductiva/_cli/cmd_tasks/download.py

```diff
@@ -0,0 +1,64 @@
+"""Download the outputs of a task by ID via CLI."""
+import argparse
+
+import inductiva
+
+
+def download_outputs(args):
+    """Download the outputs of a task by ID."""
+    task_ids = args.id
+    filenames = args.filenames
+    output_dir = args.output_dir
+
+    if output_dir is not None:
+        inductiva.set_output_dir(output_dir)
+
+    if output_dir == "":
+        print("`ouput_dir` must not be an empty string.")
+        return 1
+
+    if not task_ids:
+        print("No ID(s) specified.\n"
+              "> Use `inductiva tasks download -h` for help.")
+        return 1
+
+    task_ids = set(task_ids)
+
+    for task_id in task_ids:
+        try:
+            task = inductiva.tasks.Task(task_id)
+            task.download_outputs(filenames=filenames)
+        except inductiva.client.exceptions.ApiException as exc:
+            print(f"Error for task {task_id}:", exc)
+    return 0
+
+
+def register(parser):
+    """Register the download task command."""
+
+    subparser = parser.add_parser("download",
+                                  help="Download tasks.",
+                                  formatter_class=argparse.RawTextHelpFormatter)
+
+    subparser.description = (
+        "Download the output files produced in the context of the tasks with "
+        "the given ID(s).\nAll files are downloaded unless the --filenames"
+        " list is given, in which case only\n the indicated files for each task"
+        " are downloaded.\n"
+        "The name of the output folder can be configured "
+        "through the --output_dir option.\nFiles are downloaded to a "
+        "subdirectory named after the ID of the corresponding task.")
```

Comment:
why do  line breaks have to exist in the middle of each line and not at the end?
Makes reading it harder.

Reply:
It was to prevent long lines being show in therminal. But I will change that